### ISSUE TITLE
First attempts at data input

### ISSUE THREAD

**Issue by claireh93** [ScottishCovidResponse/SCRCIssueTracking#192]
We have created some fake data input files in HDF5 and TOML format to be read into the model to start setting up our data pipeline (available here: https://www.gla.ac.uk/research/az/boydorr/people/byname/richardreeve/ as Param.toml and ScotPop.h5)

**Comment by claireh93** [comment]
Param.toml has two parameters for beta_env and beta force (two transmission parameters), as well as their units in string form - which we should find some way to convert. Scotpop.h5 has the population data for Scotland used in some of the examples, so should be fairly straightforward to import.

**Comment by AlexRobson** [comment]
Something like this may work. 



```

using Simulation
using Unitful
using Unitful.DefaultSymbols
using Simulation.Units
using HDF5
import Pkg

p = Pkg.TOML.parsefile("Media_722505_smxx.toml")

_unit = p["params"]["unit"]
β_env = p["params"]["beta_env"]
β_force = p["params"]["beta_force"]

β_env = uparse("$(β_env) $(_unit)", unit_context = Simulation.Units)
β_force = uparse("$(β_force) $(_unit)", unit_context = Simulation.Units)

@test β_env isa Unitful.Quantity
@test β_env == 1.0/day

@test β_force isa Unitful.Quantity
@test β_force == 1.0/day


scotpop = h5open("Media_722506_smxx.h5", "r") do file
    read(file, "scotpop")
end

@test isapprox(sum(scotpop[findall(.!(isnan.(scotpop)))]), 5.45e6; rtol = 1e-3)

```
I can implement this (haven't looked at the h5 yet). Would this be a test case?

**Comment by AlexRobson** [comment]
Also, in my WIP I'm setting up DataDeps for version control. Test files like this can justifiably be added to git, but for larger data files it may be worth looking at that for reproducability.

**Comment by claireh93** [comment]
That looks great! I had a discussion with @richardreeve  the other day about DataDeps - I can't say I fully understand how it works, but it sounds really useful and we would be keen to start using it here.

**Comment by AlexRobson** [comment]
Cool - I'll make a sketch of how we could use it then.

**Comment by claireh93** [comment]
Closed in favour of #516
